src/FirebaseArduino.cpp

```diff
@@ -17,12 +17,20 @@
 #include "FirebaseArduino.h"
 
 void FirebaseArduino::begin(const char* host, const char* auth) {
+  begin(String(host), String(auth));
```

Comment:
maybe we should do the opposite, have the String& variant call the char* one?

I think that would be one less copy.